### PR TITLE
Added Companion Bank Types

### PR DESCRIPTION
**Pull request by thedist**
Added type interfaces for Companion Banks, and updated the `CompanionFeedback` interface as the `callback` function has a 2nd parameter for the feedbacks bank.

### CODE REVIEW

File: instance_skel_types.d.ts [hunk 1]
```diff
@@ -5,6 +5,51 @@ export interface CompanionSystem extends EventEmitter {}
...
+
+export interface CompanionBankPage {
+	style: 'pageup' | 'pagedown' | 'pagenum'
+}
+
+export interface CompanionBankPNG {
+	style: 'png'
+	text: string
+	size: 'auto' | '7' | '14' | '18' | '24' | '30' | '44'
+	alignment:
```
Comment (Julusian) [inline instance_skel_types.d.ts:18]: Please make a CompanionBankAlignment type for these to reduce the duplication

File: instance_skel_types.d.ts [hunk 2]
```diff
@@ -5,6 +5,51 @@ export interface CompanionSystem extends EventEmitter {}
...
+		| 'center:bottom'
+		| 'right:bottom'
+	png64?: string
+	color: number
+	bgcolor: number
+	latch: boolean
+	relative_delay: boolean
+}
+
+export interface CompanionBankPreset {
```
Comment (Julusian) [inline instance_skel_types.d.ts:45]: I'm not keen on CompanionBankPreset and CompanionBankPNG being separate types, as they are the same thing, but as the preset version wants a lot of the properties to be optional then it becomes a little messy.

I don't expect you to do anything about this right now, I may have a play myself later

**Review by Julusian — CHANGES_REQUESTED** [review]

**Comment by thedist** [comment]
Yeah make any edits you need, my reasoning for this PR is that I'm quite extensively rewriting the vMix module in Typescript, and it was giving warnings because some of my feedback callbacks utilize the bank data which is in the 2nd param, but the interface as it currently stands only defines the 1st param.

**Comment by Julusian** [comment]
I've made some changes and merged this in 529d079c8272b21c09f12ba4bdd9e2cd66da449d.

Thanks for doing the bulk of it :)

**Pull request closed without merging**